lib/sword/mets/eprints_dc_mets_file.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Sword::Mets::EprintsDcMetsFile < Sword::Mets::MetsFile
+  include Sword::Mets::EprintsDcConstants
+
+  attr_accessor :abstract,
+                :bibliographic_citation,
+                :creators,
+                :date_available,
+                :identifier_uri,
+                :publisher,
+                :title,
+                :xpath_info
+
+  def initialize(nokogiri_xml_doc)
+    super
+    @xpath_info = XPATH_INFO
+    md_wrap_xml_data_element = find_md_wrap_xml_data_element(mdtype: OTHER, other_mdtype: OTHER_MDTYPE)
+    @xml_data_element = Sword::Mets::EprintsDcXmlDataElement.new(md_wrap_xml_data_element, XPATH_INFO)
+  end
+
+  def parse
```

Comment:
Is there any case when you would want to instantiate a `Sword::Mets::EprintsDcMetsFile` but you _wouldn't_ want to call `.parse` on it right after?  If not, would it make sense to call parse in the initialize method?

(And same question MetsFile, ModsMetsFile, ModsXmlDataElement, ProquestEtdXmlDataElement, and XmlDataElement classes.)

Reply:
The design is still in flux. Depending on how I merge this functionality with the current parsing and endpoint functionality, I may end up instantiating the object without calling parse right afterwards.